src/gui/optionsdialog.ui

```diff
@@ -3054,6 +3054,13 @@ Specify an IPv4 or IPv6 address. You can specify &quot;0.0.0.0&quot; for any IPv
                     </property>
                    </widget>
                   </item>
+                  <item>
+                   <widget class="QCheckBox" name="checkSecureCookie">
+                    <property name="text">
+                     <string>Enable cookie Secure flag (requires HTTPS)</string>
```

Comment:
Perhaps the "(requires HTTPS)" is now redundant? Since the availability of the checkbox now depends on HTTPS setting.

Reply:
Or maybe not... because this setting is far away from the HTTPS settings group. Forget this.